### ISSUE TITLE
AutoCrafting RedStone Furnace

### ISSUE THREAD

**Issue by csb1020** [AppliedEnergistics/Applied-Energistics-2#966]
I end up with a java error every time I try to make the redstone furnace, I can make all the other parts but the furnace itself causes an error and doesn't craft.  No crashing or anything just hiccups.

![2015-03-03_16 12 52](https://cloud.githubusercontent.com/assets/11303834/6474753/d0ff0e54-c1c0-11e4-95d0-7e25af11c4a2.png)
![2015-03-03_16 12 56](https://cloud.githubusercontent.com/assets/11303834/6474755/d43c0b12-c1c0-11e4-8b53-c4bf357b22db.png)
![ss 2015-03-03 at 05 14 08](https://cloud.githubusercontent.com/assets/11303834/6474756/d6781042-c1c0-11e4-9864-d69bd6e27c1c.png)

**Comment by Cisien** [comment]
Pass in your full log, please. It should tell us the versions you are running of mods, and if any other issues are there

**Comment by csb1020** [comment]
I can't post the log it's too big, I'll see what I can do though to get a list of the mods we have on the server

**Comment by KlutzLament** [comment]
Post the log on http://pastebin.com/ and then post the link here.

**Comment by csb1020** [comment]
I tried that, also too big for paste bin, I've been on the server since 8am-11:37pm MST lol had today off from work and minecraft has been fun lol

**Comment by yueh** [comment]
Based on the stacktrace it seems like a corrupted pattern or recipe in general.

**Comment by csb1020** [comment]
The recipe and pattern are fine I've remade that recipe twice after getting that error in the case it was broken.  The recipe itself works, I made 7 of them, I've gone through the pattern as well step by step making sure it wasn't a part to the final recipe and it seems just the final is giving off the error.  I can try again, hopefully get a smaller log or something to post today to help you guys out with what's causing this.

**Comment by csb1020** [comment]
https://drive.google.com/file/d/0B-TQ3nAUEjcqYzlRQ3kzRHdVWFk/view?usp=sharing

**Comment by yueh** [comment]
I cannot reproduce it with AE2 and TE alone. Your pack also contains MineTweaker so I would assume there are custom recipes used, maybe something is wrong with them?

Based on exception, there is no indication of which items are causing it, just one required to craft the final product and it occured because the output of the defined pattern did not equal the acutal requested item (or no output at all). Normally indicating a corrupt pattern.

**Comment by csb1020** [comment]
Let me see if I can get the Server owner on this chat, but I've remade the recipe several times, all parts check out just not the final piece.  The Pulveriser works as well as all my other recipes I have installed into the interfaces.  So honestly I really don't know what's going on if it's on his end with anything or conflicting some how idk.  So I will see about getting him on he knows more than I do about fixing things and his mods and what he's done with MineTweaker if anything... honestly it makes no sense to me, cause I've played with AE since AE1 and I know about sometimes having a pattern that doesn't check out that's why I remade it a few times then went down the line to see if maybe I screwed up on anther recipe towards it... all work just not the final piece and only on that recipe.  I don't get it but if it's on our end I'm sure our owner will find it with your help!!!  I do appreciate it!

**Comment by grandmind** [comment]
Server owner here, the problem has been resolved. We're still not quite sure what was causing it, though. We did many things all at once in an attempt to fix the issue. The only thing that seemed very out of the ordinary was some bricks with metadata in his ME system. We removed the bricks and remade the brick pattern. We also broke and replaced with interface that had the patterns in as well as remade every pattern in the recipe, so we're not sure if it was just a problem with the bricks.
